lib/style.css

```diff
@@ -101,6 +115,8 @@ body {
 }
 
 .material .song-row .song-indicator {
+  background-image: url('ani_equalizer_white_x2.gif');
```

Comment:
For anyone wondering about this file path, it is the same as the current one
```
url('ani_equalizer_black_x2.gif');
```

But the color is changed, for some reason google also host  a white one so we can just point to that.  :smile: